src/main/flight/mixer_init.c

```diff
@@ -50,7 +50,8 @@ PG_RESET_TEMPLATE(mixerConfig_t, mixerConfig,
     .mixerMode = DEFAULT_MIXER,
     .yaw_motors_reversed = false,
     .crashflip_motor_percent = 0,
-    .crashflip_expo = 35
+    .crashflip_expo = 35,
+    .linear_mixer = true
```

Comment:
I'd consider this very experimental and would think it should be OFF by default. Otherwise people who weren't planning on testing this will be surprised by a changed flight behavior. After testing and confirmation we could revisit whether it should be the default. But that's more likely to be in the 4.4 timeframe.

Reply:
Yes, I agree, definitely should not enable by default.
It's highly experimental and needs a lot of testing.
I envisage it causing issues for cinematic and proximity flying, since any PID input will change thrust, even smallish PID changes; the effect is maximal at cruising throttle.

Reply:
I agree too